docs/contribute.rst

```diff
@@ -182,6 +182,17 @@ Here's the overview of core classes::
     │   └── GuestTestcloud
     └── Run
 
+Attributes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Object hierarchy is following: Run -> Plans -> Steps -> Plugin, where the Run is
+on the top of this hierarchy. The objects have the ``parent`` attribute, that is
+pointing on a parent from which the current object inherits from.
```

Comment:
`Plan` does not inherit from `Run`, they are included. Imagine `Run` as a container of plans. Let's use "contain" here to prevent possible confusion.